_rules/role-attribute-valid-value-674b10.md

```diff
@@ -80,23 +71,34 @@ Further reading:
 This `role` attribute contains one token, and this token is a valid [WAI-ARIA role][].
 
 ```html
-<input type="text" role="textbox" />
+<input type="text" role="searchbox" />
```

Comment:
```suggestion
<label for="srch">Search</label>
<input type="text" role="searchbox" />
```

Shouldn't we somehow imply in the examples what the visual presentation is?

Reply:
Agreed, but the `input` needs an `id="srch"` attribute

Reply:
yes, updated.